dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/productinfo/ProductInfo.java

```diff
@@ -225,6 +227,41 @@ public String getProperty(String key) {
         return value;
     }
 
+    /**
+     * Helper method for determining if product is EARLY_ACCESS edition
+     */
+    public boolean isBeta() {
+        boolean isBeta;
+        String productEdition = this.getEdition();
+        if (productEdition == null) {
+            isBeta = false;
+        } else {
+            isBeta = productEdition.equals(EARLY_ACCESS);
+        }
+        return isBeta;
+    }
+
+    /**
+     * Sets the com.ibm.ws.beta.edition JVM property to true if the Liberty version is EARLY_ACCESS,
+     * otherwise sets it to false.
+     */
+    public static void isAnyProductBeta() {
```

Comment:
The method name isAnyProductBeta seems odd.  I was expecting it to return something. Its purpose seems to be to ensure the java property is set.